pkgs/build-support/cc-wrapper/default.nix

```
@@ -371,11 +371,11 @@ stdenv.mkDerivation {
         echo "-isystem $dir" >> $out/nix-support/libcxx-cxxflags
       done
     ''
-    + optionalString (libcxx.isLLVM or false) (''
+    + optionalString (libcxx.isLLVM or false) ''
       echo "-isystem ${lib.getDev libcxx}/include/c++/v1" >> $out/nix-support/libcxx-cxxflags
       echo "-stdlib=libc++" >> $out/nix-support/libcxx-ldflags
-      echo "-lc++abi" >> $out/nix-support/libcxx-ldflags
-    '')
+      echo "-l${{ libcxxabi = "c++abi"; libcxxrt = "cxxrt"; }.${libcxx.cxxabi}}" >> $out/nix-support/libcxx-ldflags
```

```suggestion
      echo "-l${libcxx.cxxabiLib.libName}" >> $out/nix-support/libcxx-ldflags
```
Can we do something like that?

---

I realised that, apart from this, nothing else was using the string `libcxx.cxxabi`, so I got rid of it, and then renamed what was previously `libcxx.cxxabiLib` to `libcxx.cxxabi`, so now we can do `libcxx.cxxabi.libName` (as `libcxx.cxxabi` is the package now, not a string).